Fixes typo with example in README.

lazarillo opened the pull request:
Typo in the README file so that the example does not work.

marcoceppi reviewed (changes requested):
Thanks so much for the pull request! Indeed that example did not work but I think there's one additional change needed to make this example function properly.

LGTM once that change is in

lazarillo commented on README.md:
```diff
@@ -59,7 +59,7 @@ def test_sqs_deliver(aws_sqs):
    # respecting scopes.
    #
-   boto3.Session().client('sqs')
+   Session().client('sqs')
```
Yes, I will do that.

lazarillo commented:
@marcoceppi Thanks for your suggestion!  I added that and updated the repo on my fork.

Hopefully this gave me a few brownie points: I do not understand how I should structure my project, and I [put my question in an issue](https://github.com/cloud-custodian/pytest-terraform/issues/37#issue-770184739).  :)

I am hoping it is an easy question to answer.

marcoceppi reviewed (approved):
This looks great, thank you! 👍 I'll follow up with you in the other thread with regards to your question
